pkg/routing/table.go

```diff
@@ -40,43 +39,26 @@ type Table interface {
 
 	// Count returns the number of RoutingRule entries stored.
 	Count() int
+
+	// CollectGarbage checks all the stored rules, removes and returns ones that timed out.
+	CollectGarbage() []Rule
```

Comment:
What's the reasoning behind returning `Rule` instead of `RouteDescriptor`?

Reply:
@evanlinjin routing table stores separate rules, not descriptors. descriptors are part of a route group which stands a bit above the table, table itself is rather simple